components/order-menu/drink-menu/index.js

```diff
@@ -0,0 +1,10 @@
+export default function Drinks() {
+  return (
```

Comment:
I would recommend to make this component a <ul> with it's <li> and not importing the <li> into OrderList.  Same goes for Maindishes()